includes/components/class-user.php

```diff
@@ -403,4 +407,36 @@ public function alter_site_footer_block( $template ) {
 			]
 		);
 	}
+
+	/**
+	 * Alters authour link.
+	 *
+	 * @param string $link Author page link.
+	 * @param int    $user_id User ID.
+	 * @param string $username Username.
+	 * @return string
+	 */
+	public function alter_author_link( $link, $user_id, $username ) {
```

Comment:
This doesn't seem to be required, the end link will redirect to the user or vendor page anyway (?)